app/models/bo/client_id_lookup.rb

```diff
@@ -265,6 +266,69 @@ def call_with_retry settings, message_options
       return response
     end
 
+    def fetch_disability_verifications one_off: false
+      return unless @config.disability_verification_cuid.present?
+      msg = "Fetching disability verifications"
+      Rails.logger.info msg
+      @notifier.ping msg if send_notifications && msg.present?
+      settings = {
+        url: "#{@config.url}?wsdl=1&cuid=#{@config.disability_verification_cuid}",
+        method: :disability_lookup,
+      }
+      message_options = {
+        touch_point_id: @config.disability_touch_point_id,
+        touch_point_question_id: @config.disability_touch_point_question_id,
+      }
+      if one_off
```

Comment:
Tempting to suggest calling `call_with_retry` here with a new parameter `retries` - it just looks odd to build up the `settings` and `message_options` objects, and then tear them apart again.

Reply:
re-written with a `call_once` method to use the same structure as `call_with_retry` but not do the retries.